vnext/Microsoft.ReactNative/TurboModulesProvider.cpp

```diff
@@ -254,13 +263,13 @@ class TurboModuleImpl : public facebook::react::TurboModule {
               auto argReader = winrt::make<JsiReader>(runtime, args, count);
 
               // prepare output value
-              auto argWriter = winrt::make<JsiWriter>(runtime);
+              auto writer = winrt::make<JsiWriter>(runtime);
 
               // call the function
-              method(argReader, argWriter);
+              method(argReader, writer);
 
-              // return the result
-              return argWriter.as<JsiWriter>()->MoveResult();
+              // set the return value
```

Comment:
>// set the return value [](start = 14, length = 23)

I think my comment was better :)

Reply:
I can just undo this... since you already checked in the real change.

Reply:
Actually, the comment is silly.  You could put this comment on any return statement anywhere.